website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,222 @@
+
+## Assumptions
+
+This guides makes the following assumptions:
+
+- You are familiar with the [General Upgrade Process](/docs/upgrading/instructions/general-process).
+- You have the ability to run Consul CLI commands. 
+- If ACLs are in use then you possess a token with at least `operator:read` permissions.
+
+## Considerations
+
+The main breaking changes in Consul Enterprise 1.10 that need special handling during upgrading are
+the licensing changes outlined on the [Specific Version Details](/docs/upgrading/upgrade-specific#consul-1-10-0)
+page. There are notes on that page about other changes that might cause issues during an upgrade as well.
+You can find more granular details in the [licensing FAQ](/docs/enterprise/license/faq) as well as the full 
+[changelog](https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#1100-june-22-2021).
+Looking through these changes prior to upgrading is highly recommended.
+
+## Procedures
+
+~> When any of the steps below call for upgrading components, we assume you follow the normal safe upgrade
+procedure you normally use for upgrades. These instructions require licensing related configuration changes
+needed to upgrade to 1.10, and also require intermediate version upgrades to ensure forward compatibility. 
+Refer to our documentation for the [basic server upgrade process](/docs/upgrading/instructions/general-process), 
+and for [autopilot assisted upgrades](https://learn.hashicorp.com/tutorials/consul/upgrade-automation)
+for more information.
+
+**1.** Retrieve the datacenter's current license by executing the following CLI command:
+
+```shell
+consul license get -signed > consul.hclic
+```
+
+  This will save the license to a file in the local directory named `consul.hclic`. Having this 
+  will be useful when preparing the license for the 1.10 upgrades later on in the process.
+
+**2.** Take a snapshot of the cluster by running:
+
+```shell
+consul snapshot save original.snap
+```
+
+  This snapshot should not be needed but if something were to go wrong, having
+  the snapshot would make it possible to restore the previous state.
+
+**3.** Determine if ACLs are enabled by running the following CLI command.
+
+```shell
+consul info | grep "acl ="
+````
+
+  If you receive output like either of the following then ACLs are enabled:
+
+```text
+Error querying agent: Unexpected response code: 403 (Permission denied)
+```
+
+```text
+        acl = enabled
+
+```
+
+  Select the correct tab below based on whether ACLs are enabled or disabled or if you
+  are operating Consul within Kubernetes.
+
+<Tabs>
+<Tab heading="ACLs Enabled">
+
+**4.** Upgrade all server agents to the latest 1.8.x or 1.9.x release.
+
+**5.** Upgrade all client agents to the latest 1.8.x or 1.9.x release.
+
+**6.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to the latest 1.8.x or 1.9.x release.
+
+**7.** Take a snapshot of the upgraded cluster by running:
```

Comment:
nit: where should they run that, on a server? Should we mention checking disk space before running it? That gets technical but maybe at least a nod towards the fact this could write a large file to disk might be useful?

Reply:
I added information that said they could be run from pretty much anywhere but if you don't execute from a server/client then you will need to pass other options to the save command (and linked out to it). Also added a second note about disk requirements.